x-pack/plugins/watcher/common/types/status_types.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Moment } from 'moment';
+import type * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+
+import { ACTION_STATES, WATCH_STATES, WATCH_STATE_COMMENTS } from '../constants';
+
+export interface ActionStatusUpstreamJson {
```

Comment:
Based on your PR description, it seems like the intention was to move away from upstream/downstream concepts. Should this be renamed?